R/model_eval.R

```diff
@@ -145,21 +145,21 @@ evaluate_binary_ <- function(df, actual_val_col, pred_prob_col, threshold = "f_s
 
 #' Non standard evaluation version of evaluate_regression_
 #' @param df Data frame
-#' @param actual_val Column name for actual values
 #' @param fitted Column name for predicted values
```

Comment:
Should fitted renamed to pred_val? That sounds more in line with the names we use for output columns.